lib/database/repositories/LhcPeriodStatisticsRepository.js

```diff
@@ -25,6 +36,29 @@ class LhcPeriodStatisticsRepository extends Repository {
     constructor() {
         super(LhcPeriodStatistics);
     }
+
+    /**
+     * Returns a specific entity.
+     *
+     * @param {QueryBuilder} [queryBuilder] the find query (see sequelize findAll options) or a find query builder
+     * @returns {Promise<Object>|null} Promise object representing the full mock data
+     */
+    async findOne(queryBuilder = dataSource.createQueryBuilder()) {
+        queryBuilder.includeAttribute(...DISTINCT_ENERGIES_IN_STATISTICS_ATTRIBUTE);
```

Comment:
This do not belong here: you expect find one to fetch what you pass as query builder, not to take initiatives